help/legacy/_posts/2019-10-25-legacy.md

```diff
@@ -0,0 +1,73 @@
+# Legacy Services
```

Comment:
Missing the header, so it doesn't render correctly. Also, I'm guessing from the first line that you want this to be order=1 and then bump down all the other posts?

Reply:
Oh, and same comment about the text duplication, if you choose to go the dropdown route